### PR TITLE
Make geckolib possible to build on stable Rust

### PR DESCRIPTION
**Pull request by SimonSapin**
More for https://github.com/servo/servo/issues/11815

r? @nox

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [x] These changes do not require tests because _____

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12263)

### CODE REVIEW

File: ports/geckolib/Cargo.toml [hunk 1]
```diff
@@ -36,15 +36,15 @@ lazy_static = "0.2"
 libc = "0.2"
 log = {version = "0.3.5", features = ["release_max_level_info"]}
 num_cpus = "0.2.2"
-selectors = {version = "0.6", features = ["unstable"]}
+selectors = "0.6"
```
Comment (Manishearth) [inline ports/geckolib/Cargo.toml]: We should probably add selectors/unstable to the features list?

Comment (SimonSapin) [inline ports/geckolib/Cargo.toml]: Do you mean to the `servo` feature? Yes, done.

**Comment by Manishearth** [comment]
LGTM

**Comment by bholley** [comment]
I need to base a PR on this. @bors-servo p=1

**Comment by jdm** [comment]
@bors-servo: retry
* #11631

**Comment by SimonSapin** [comment]
@bors-servo retry

* #11703

**Comment by jdm** [comment]
@bors-servo: retry
* #11703

**Pull request merged**